Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:stable
```

Comment:
Actually I am not sure why I did not think of this before, but whenever I create a Dockerfile to create a Docker image for a Java service I just have the Dockerfile pull from a Java Docker image from the official Java Docker repository and then everything is already setup for me.  Maybe we can do the same here and just have this Dockerfile pull from a Go (Golang) Docker image from the official Go (Golang) Docker repository and then we can also set the tag to match the version of Go that we want to run.  For instance we can go with one of the following:

- `FROM golang:1.18.1` or just `FROM golang:1.18` as I do not think that there will be any issues if the 1.18 tag gets updated when the minor version number gets updated
- `FROM golang:1.17.9` or just `FROM golang:1.17` if you want to have the latest 1.17 version
- `FROM golang:1.17.7` if you want to stick to the 1.17.7 version

From what I can tell all of these Docker images are based off of the Debian bullseye release so they will all be running Debian Linux when they are running as Docker containers.  For more information on all of the Go (Golang) Docker images you can check out the official Go (Golang) Docker repository here:

https://hub.docker.com/_/golang

Reply:
@jbvandefliert That's perfect, thanks so much!